docs/index.md

```diff
@@ -1,148 +1,125 @@
+the tools to make _you_ a more productive developer by reducing
+drudgery and codifying standards.
+
+## Using Atomist
+
+Atomist helps you gain visibility and control over your software
+delivery process.  You can use Atomist's out-of-the-box automations or
+you can write your own automations.
+
+### Atomist automations
+
+Invite the Atomist bot into your Slack team and use the built-in
+automations.  The bot sends you relevant, actionable messages that
+enable you to control your development flow from Slack, from project
+creation to deployment to monitoring.
+
+### Custom automations
+
+You can customize Atomist's built-in automations or write wholly new
+automations.
+
+Atomist provides an automation API with integrations for your
+developer tools and runtime platforms, native Slack notifications and
+commands, and an open source client and command-line interface.
+
+Custom automations can be used to further streamline development:
+
+-   Notify team members about events such as pushes, pull requests,
+    build failures, crash looping Kubernetes pods and more.
+-   Create custom bot commands for fun and profit. Search
+    StackOverflow from Slack or exchange banter with other teams -
+    it's up to you.
+-   Automate your release process - rather than delegating it to your
+    build system - so you can release with a single button press in
+    Slack.
+-   Merge pull requests automatically as soon as the required checks
+    and reviews pass.
+-   Auto-close issues when a fix is deployed to production.
+-   Roll back automatically: identify a commit that caused a
+    regression in production, revert the commit, and release the "new"
+    version
+
+## Concepts
+
+Atomist is all about making you more productive by helping you to
+automate away all the things that slow you down.  The following
+concepts describe the pieces Atomist provides to make it easy for you
+to create automations that reduce drudgery and distractions.
+
+### Development automation platform
+
+Atomist's development automation platform is powered by a service that
+ingests and correlates events from your software development flow.  At
+the heart of the service is a single coherent model of your
+development flow: code, people, and processes.  You query and mutate
+that model using [GraphQL][gql] when you write your own automations.
+
+[gql]: http://graphql.org/ (GraphQL)
+
+### Automation client
+
+Code in automations is written in Atomist automation clients, which
+interact with the Atomist platform using primarily GraphQL.  Each
+client is itself a server which hosts automations that can be invoked
+via the Atomist bot, Slack buttons, or when events occur.  A client
+can host any number of automations, and can be hosted wherever the
+author likes: locally during testing, inside a corporate firewall, or
+on a public cloud or PaaS.
+
+### Commands
+
+Commands perform actions when invoked by users by sending the Atomist
```

Comment:
Commands perform actions when invoked by users ~~by~~ sending the Atomist
bot messages, click **ing** buttons in Slack, or using the Atomist CLI.